style.css

```diff
@@ -8,15 +8,16 @@ html,body{
     --black: #292524;
     --greyOne: #44403c;
     --greyTwo:#57534e;
+    --off-teal:#f0fdfa;
     --teal:#2dd4bf;
     --off-white:#fafaf9;
     --alert-green:#d9f99d;
     --alert-blue: #bfdbfe;
     --alert-yellow:#fde68a;
     --alert-red: #fecaca;
-    --dark-teal: #0d9488;
+    --dark-teal: #115e59;
```

Comment:
You may use generic names for colour so that you can use in different components too.